src/chesster.js

```diff
@@ -970,3 +972,87 @@ function(bot, message) {
     }
 });
 
+/* subscriptions */
+
+chesster.hears({
+    middleware: [],
+    patterns: ['^tell'],
+    messageTypes: ['direct_message']
+},
+function(bot, message) {
+    var deferred = Q.defer();
+    bot.startPrivateConversation(message, function (response, convo) {
+        subscription.processTellCommand(chesster.config, message).then(function(response) {
+            convo.say(response);
+            deferred.resolve();
+        }).catch(function(error) {
+            convo.say("I'm sorry, but an error occurred processing this subscription command");
+            deferred.reject(error);
+        });
+    });
+    return deferred.promise;
+});
+
+chesster.hears({
```

Comment:
Good idea. I like it.  I will switch these two commands to using that format. I will use subscription list
subscription remove (id).
I still like the tell version of the command as I think that's easier to remember and understand for non technical users.